features/features.go

```diff
@@ -409,14 +385,6 @@ var (
 					enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
 					mustRegister()
 
-	FeatureGateDisableKubeletCloudCredentialProviders = newFeatureGate("DisableKubeletCloudCredentialProviders").
-								reportProblemsToJiraComponent("cloud-provider").
-								contactPerson("jspeed").
-								productScope(kubernetes).
-								enhancementPR("https://github.com/kubernetes/enhancements/issues/2395").
-								enableIn(configv1.Default, configv1.DevPreviewNoUpgrade, configv1.TechPreviewNoUpgrade).
-								mustRegister()
-
```

Comment:
Removed in 1.33: https://github.com/kubernetes/kubernetes/blob/7cd7fea30d054efe4f2cc60073a29d2c618258ec/CHANGELOG/CHANGELOG-1.33.md?plain=1#L366

@JoelSpeed should we drop it in 4.20?

Reply:
I already have a PR open to drop this, but yes, whoever of us gets there first